library/core/src/main/java/com/google/android/exoplayer2/text/CaptionStyleCompat.java

```diff
@@ -67,7 +67,9 @@
    * Default caption style.
    */
   public static final CaptionStyleCompat DEFAULT = new CaptionStyleCompat(
-      Color.WHITE, Color.BLACK, Color.TRANSPARENT, EDGE_TYPE_NONE, Color.WHITE, null);
+          Color.WHITE, Color.BLACK,
+          /* Transparent Blue */Color.BLUE & (0x00FFFFFF),
```

Comment:
The Default Window Color from caption manager in some devices that i tested was this color. If we need to apply user styling, we need to know if the user has modified this value from the default or not. So , we need to change the default value.  LIne 325 of SubtitlePainer.java
I'm not sure if this is a bug in some devices or in AOSP code. i.e instead of opaque black. its  transparent.

Reply:
As above, this class is not CEA specific. You can't change the default to MONOSPACE just because it fixes CEA-608, because it will affect how other caption types are rendered. If we need to be able to specify that CEA-608 uses monospace fonts, then that information needs to be hardcoded somewhere in CEA-608 specific code. For example by setting styling information on each `Cue`. The text package then needs to be able to handle that information in the correct way.